packages/client/e2e/utils.ts

```diff
@@ -3,6 +3,9 @@ import { expect, Locator, WorkerInfo } from "@playwright/test"
 export const isOpenFin = (workerInfo: WorkerInfo) =>
   workerInfo.project.name === "openfin"
 
+export const isResponsive = (workerInfo: WorkerInfo) => 
```

Comment:
ok, for now .. but we should add `openfinResponsive` project too .. and then `isOpenFin` will need to change as well, obviously .. this is not an academic point, I ran up OpenFin without maximise and the viewport settings, as I was trying to remove that kind of thing (and use of `workerInfo`) from the actual spec file .. it failed as the smaller window triggered the responsive Tab -> Dropdown 😐